runtime/services/activation/activation.cc

```diff
@@ -52,7 +50,13 @@ class Activation : public ClientCallback {
     }
 
     int id = rand() % 4;
-    prepareWavPlayer(filenames[id], "system", true);
+    char audio_hold[PROP_VALUE_MAX];
+    bool hold = true;
+    property_get("persist.act.audio.holdcon", (char*)audio_hold, "1");
+    if (strcmp(audio_hold, "0") == 0) {
+      hold = false;
+    }
+    prepareWavPlayer(filenames[id], "system", hold);
```

Comment:
Tips: #154 refactors the activation, and see https://github.com/yodaos-project/yodart/pull/154#issuecomment-450166377 that I commented the design guide of activation.